src/main/python/core/global_manager.py

```diff
@@ -265,18 +265,18 @@ def grch_build(self):
         """
         Getter for setter grch_build
 
-        :return: integer
+        :return: int
         """
         return self._grch_build
 
     @grch_build.setter
     def grch_build(self, value):
         """
-        Singleton property grch_build, to set the globally available GRCh build
-        parsed from the config.
+        Singleton property grch_build,
+        to set the GRCh build present in the parsed input file.
```

Comment:
False, this value is set after evaluating both the config and input file GRCh builds to be used in imputing and predicitng.

Reply:
> `@grch_build.setter` -> sets final grch build (fix was invalid)

Correct

> `@config_grch_build.setter` -> "to set the GRCh build present in the config"

Correct

> `@vep_version.setter` -> "VEP version present in the parsed input file"

Incorrect, this is the same as grch_build.setter, but for the VEP version.

> `@config_vep_version.setter` -> "VEP version present in the config"

Correct.

Reply:
Aaaah, then I fixed it the wrong way around. Thanks!

Reply:
Commit is added which corrects it: https://github.com/molgenis/capice/pull/55/commits/c9427dfc84411493d3ce94beda07731a3c519e74